test-case/check-audio-equalizer.sh

```diff
@@ -44,103 +45,90 @@ func_test_eq()
     local id=$1
     local conf=$2
 
-    [ ! -f $conf ] && dloge "$conf not exist" && exit 1
-
     dlogc "sof-ctl -Dhw:$sofcard -n $id -s $conf"
-    sof-ctl -Dhw:$sofcard -n $id -s $conf
-    if [ $? -ne 0 ]; then
+    sof-ctl -Dhw:"$sofcard" -n "$id" -s "$conf" || {
         dloge "Equalizer setting failure with $conf"
         return 1
-    fi
-    # test with aplay
-    dlogc "aplay -D$dev -f $fmt -c $channel -r $rate /dev/zero -d $duration"
-    aplay -D$dev -f $fmt -c $channel /dev/zero -d $duration
-    if [ $? -ne 0 ]; then
+    }
+
+    dlogc "$cmd -D $dev -f $fmt -c $channel -r $rate -d $duration $dummy_file"
+    $cmd -D "$dev" -f "$fmt" -c "$channel" -r "$rate" -d "$duration" "$dummy_file" || {
         dloge "Equalizer test failure with $conf"
```

Comment:
@aiChaoSONG already update those type code to `die` function, but it is not refer you code